Dockerfile

```diff
@@ -1,31 +1,28 @@
 FROM debian:latest
 
-RUN apt-get update && apt-get install -y  gcc \
-					make \
-					vim \ 
-					gdb \  
-					bison flex \
-					man \
-					ftp
-COPY . /wu-ftpd
+RUN apt-get update && \
+    apt-get install -y bison \
+                       flex \
+                       ftp \
+                       gcc \
+                       gdb \
+                       make \
+                       man \
+                       vim
 
-WORKDIR /wu-ftpd/wu-ftpd-2.6.0/
-RUN ./configure
-RUN mkdir /usr/man/
-RUN mkdir /usr/man/man8 && cp ./doc/ftpd.8 /usr/man/man8
-RUN mkdir /usr/man/man1 && cp ./doc/ftpcount.1 /usr/man/man1
-RUN mkdir /usr/man/man5 && cp ./doc/ftpaccess.5 /usr/man/man5
-RUN sed -i 's/\t=\t{/\t{/g' src/ftpcmd.y
```

Comment:
@neverrend can you provide a reference for this fix?

Reply:
Reference?

Reply:
Yes, for where you got this fix. Or at least a comment for what it does.

Reply:
Yes, for where you got this fix. Or at least a comment for what it does.

Reply:
Yes, for where you got this fix. Or at least a comment for what it does.

Reply:
Oooh ok I got it right here, they didnt explicitly say use that `sed` line I wrote but that's the best way I could come up with based on the conversation: 
https://lists.gnu.org/archive/html/help-bison/2004-04/msg00006.html